conans/test/functional/configuration/registry_test.py

```diff
@@ -127,3 +127,23 @@ def insert_test(self):
                          Remote("repo2", "url2", True),
                          Remote("conan.io", "https://server.conan.io", True),
                          Remote("repo3", "url3", True)])
+
+    def test_remote_none(self):
```

Comment:
@memsharded is this valid? I can set remote's URL to None, but does it make sense?

Reply:
Well, you can if you use the internals, like ``registry.add()`` but the behavior will be breaking somewhere, if the value is attempted to use.

It is not necessary to handle and to test every possible argument to be valid, it becomes a bit bloated with little benefit.